Description: Fix syntax errors in example files.
Bug-Debian: http://bugs.debian.org/580271
Author: Arthur de Jong <adejong@debian.org>

--- a/examples/chess.py
+++ b/examples/chess.py
@@ -23,7 +23,7 @@
                 self.term.process_list (self.before)
                 self.term.process_list (self.after)
 		self.last_computer_move = ''
-        def read_until_cursor (self, r,c)
+        def read_until_cursor (self, r,c):
             while 1:
                 self.child.read(1, 60)
                 self.term.process (c)
@@ -41,7 +41,7 @@
                 read_until_cursor (19,60)
 		#self.child.expect ('\[19;60H')
 		self.child.sendline (move)
-		print 'do_move' move
+		print 'do_move', move
 		return move
 
 	def get_first_computer_move (self):
--- a/examples/fix_cvs_files.py
+++ b/examples/fix_cvs_files.py
@@ -24,8 +24,8 @@
 
 def is_binary (filename):
 
-    """Assume that any file with a character where the 8th bit is set is
-    binary. """
+	"""Assume that any file with a character where the 8th bit is set is
+	binary. """
 
 	fin = open(filename, 'rb')
 	wholething = fin.read()
@@ -37,9 +37,9 @@
 
 def is_kb_sticky (filename):
 
-    """This checks if 'cvs status' reports '-kb' for Sticky options. If the
-    Sticky Option status is '-ks' then this returns 1. If the status is
-    'Unknown' then it returns 1. Otherwise 0 is returned. """
+	"""This checks if 'cvs status' reports '-kb' for Sticky options. If the
+	Sticky Option status is '-ks' then this returns 1. If the status is
+	'Unknown' then it returns 1. Otherwise 0 is returned. """
 
 	try:
 		s = pexpect.spawn ('cvs status %s' % filename)
@@ -59,7 +59,7 @@
 
 def cvs_admin_kb (filename):
 
-    """This uses 'cvs admin' to set the '-kb' sticky option. """
+	"""This uses 'cvs admin' to set the '-kb' sticky option. """
 
 	s = pexpect.run ('cvs admin -kb %s' % filename)
 	# There is a timing issue. If I run 'cvs admin' too quickly
@@ -68,8 +68,8 @@
 	
 def walk_and_clean_cvs_binaries (arg, dirname, names):
 
-    """This contains the logic for processing files. This is the os.path.walk
-    callback. This skips dirnames that end in CVS. """
+	"""This contains the logic for processing files. This is the os.path.walk
+	callback. This skips dirnames that end in CVS. """
 
 	if len(dirname)>3 and dirname[-3:]=='CVS':
 		return
--- a/examples/ssh_tunnel.py
+++ b/examples/ssh_tunnel.py
@@ -56,7 +56,7 @@
             print 'restarting tunnel'
             start_tunnel ()
             time.sleep (11)
-	        print 'tunnel OK'
+            print 'tunnel OK'
         else:
             # print 'tunnel OK'
             time.sleep (7)
